edx/edx-app-ios
# Registration Accessibility

### Description

[MA-2721](https://openedx.atlassian.net/browse/MA-2721)
[MA-2722](https://openedx.atlassian.net/browse/MA-2722)
[MA-2723](https://openedx.atlassian.net/browse/MA-2723)
[MA-2724](https://openedx.atlassian.net/browse/MA-2724)
[MA-2725](https://openedx.atlassian.net/browse/MA-2725)
[MA-2726](https://openedx.atlassian.net/browse/MA-2726)
Fix for [MA-2764](https://openedx.atlassian.net/browse/MA-2764)

Accessibility issues on the registration screen have been implemented within this PR. Also, a fix for MA-2764 has been added in this PR.

### Notes

Please refer to the Jira tickets for detailed description of the functionalities implements within this PR.

### How to test this PR

1. Navigate to the Registration screen
2. Turn on accessibility
3. Make sure all components are reliant to the a11y standards

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
